network/network.go

```diff
@@ -38,6 +41,14 @@ var (
 	_ NetworkInterface = (*Network)(nil)
 )
 
+func init() {
+	log.Logger = log.With().Caller().Logger()
+	// prettify if terminal is a console
+	if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
Will this always be a terminal? When would it be false?

Reply:
its false if its running as tests